.evergreen/config.yml

```diff
@@ -974,18 +974,26 @@ functions:
           ${PREPARE_SHELL}
 
           cd ${DRIVERS_TOOLS}/.evergreen/csfle
-          source activate_venv.sh || :
+
+          # only run this if virtualenv is installed
+          if ${PYTHON3_BINARY} -m venv ./venv; then
+            . ./activate_venv.sh
+
+            # TODO (GODRIVER-2239): Stabilize this pip install with a non-forked version of PyKMIP in
+            pip install git+https://github.com/kevinAlbs/PyKMIP.git@expand_tls12_ciphers
+          else
+            echo "error: No module named venv"
+          fi
     - command: shell.exec
       params:
         background: true
         script: |
           cd ${DRIVERS_TOOLS}/.evergreen/csfle
-          . ./activate_venv.sh
-
-          # TODO (GODRIVER-2239): Stabilize this pip install with a non-forked version of PyKMIP in
-          pip install git+https://github.com/kevinAlbs/PyKMIP.git@expand_tls12_ciphers
-
-          ./kmstlsvenv/bin/python3 -u kms_kmip_server.py --port 5698
+          if [ "Windows_NT" = "$OS" ]; then
+            kmstlsvenv/Scripts/python.exe -u kms_kmip_server.py --port 5698
+          else
+            ./kmstlsvenv/bin/python3 -u kms_kmip_server.py --port 5698
+          fi
```

Comment:
Is it necessary to use the Python 3 executable from the venv? Most scripts use the Evergreen expansion `${PYTHON3_BINARY}` which is set to a different value on Windows vs Linux vs macOS builds. Could we replace these two different commands with
```bash
${PYTHON3_BINARY} -u kms_kmip_server.py --port 5698
```
?

Reply:
It is necessary, the python executable created through activate_venv.sh has dependencies that the pre-installed python3 binary does not have. This is what happens when using the pre-install exe:

```
Administrator@EC2AMAZ-KTJHJ9U ~/drivers-evergreen-tools/.evergreen/csfle
$ C:/python/Python38/python.exe -u kms_kmip_server.py --port 5698
Traceback (most recent call last):
  File "kms_kmip_server.py", line 6, in <module>
    from kmip.services.server import KmipServer
ModuleNotFoundError: No module named 'kmip'
```

Reply:
@matthewdale Most of the Windows x64 tasks pass but we do get some failures due to [this prose test](https://github.com/mongodb/mongo-go-driver/blob/master/mongo/integration/client_test.go#L654): https://evergreen.mongodb.com/task/mongo_go_driver_tests_42_plus_zlib_zstd_support__version~latest_os_ssl_40~windows_64_go_1_17_test_standalone_auth_ssl_zlib_compression_patch_404a99a20576ef7d86f62910917c53baf0dd9319_62be1a701e2d177d7c214011_22_06_30_21_49_37##comparehashes=404a99a20576ef7d86f62910917c53baf0dd9319&threads=maxonly

I _think_ they are unrelated to this ticket, though. @benjirewis could you opine?

Reply:
They should be totally unrelated to this ticket, feel free to ignore. Opened GODRIVER-2483.